src/settings/MappingProfiles/detailsSections/edit/InstanceDetailsSections/AdministrativeData.js

```diff
@@ -42,6 +42,13 @@ export const AdministrativeData = ({
   getRepeatableFieldAction,
   okapi,
 }) => {
+  const validateDatepickerValueField = useCallback(
```

Comment:
Did you mean `validateDatepickerFieldValue`?

Reply:
Done